lib/tasks/import/service_links_from_csv.rake

```diff
@@ -14,8 +14,8 @@ namespace :import do
     imported = 0
 
     csv.each do |row|
-      slug = row["slug"]&.strip
-      url = row["url"]&.strip
+      slug = row[0]&.strip
+      url = row[1]&.strip
```

Comment:
I feel using column position is somewhat risky - far too easy to switch columns around.  Using headers seems much safer.  

Microsoft Excel puts/keeps this byte string at the start of every line `\xEF\xBB\xBF` so I'm concerned that we are storing those characters, as they are not being removed from the actual column value.  Which may have undesirable consequences and tricky to track bugs.  Thankfully, Google Sheets doesn't seem to do this!  

We could strip this stuff out `.gsub("\xEF\xBB\xBF".force_encoding("UTF-8"), "")` for example.  But I wonder if we could just 'encourage' folks to use Google Sheets to do the conversion?

Reply:
Looking into this a bit further, it seems that only the first row contains `\xEF\xBB\xBF` and not all rows.  So, there should be no issues with saving this stuff.  Assuming we use headers.

Reply:
@Rosa-Fox I'll approve this and leave the choice of `row[0]` or `row["slug"]` to you.